test/suites/ami/suite_test.go

```diff
@@ -282,6 +282,7 @@ var _ = Describe("AMI", func() {
 			actualUserData, err := base64.StdEncoding.DecodeString(*getInstanceAttribute(pod.Spec.NodeName, "userData").UserData.Value)
 			Expect(err).ToNot(HaveOccurred())
 			Expect(string(actualUserData)).To(ContainSubstring("kube-api-qps = 30"))
+			Expect(string(actualUserData)).To(ContainSubstring("memory.available = 300Mi"))
```

Comment:
Can we validate that we this value is assigned to the eviction soft section? As it stands, this check would pass if we passed the same key to eviction hard

Reply:
Added the E2E integration test as requested, please review at your convenience, thx!